site/docs/4.1/components/dropdowns.md

```diff
@@ -595,7 +595,9 @@ Add `.disabled` to items in the dropdown to **style them as disabled**.
 
 ## Menu alignment
 
-By default, a dropdown menu is automatically positioned 100% from the top and along the left side of its parent. Add `.dropdown-menu-right` to a `.dropdown-menu` to right align the dropdown menu.
+By default, a dropdown menu is automatically positioned 100% from the top and along the left side of its parent. 
+When positioned horizontally add `.dropdown-menu-right` to a `.dropdown-menu` to right align the menu or `.dropdown-menu-center` to center.
```

Comment:
@mdo This is a bit hard to understand, I'd rewrite it as:

> Apply `.dropdown-menu-right` or `.dropdown-menu-center` as a modifier to `.dropdown-menu` to right or center align the menu on a dropdown or dropup. And apply `.dropdown-menu-bottom ` or `.dropdown-menu-center` as a modifier to `.dropdown-menu` to bottom or center align the menu on a dropright or dropleft.

What do you think?

Reply:
I tried as hard as I could, but I'm not a native speaker =)

Reply:
@vepanimas don't worry, me neither, that's why I am asking @mdo 😝

Reply:
Maybe @XhmikosR can help us too 😄 or @patrickhlauke

Reply:
@mdo: can you chime in here? Otherwise this can be merged AFAIK.